storage/client/client.go

```diff
@@ -1437,13 +1462,85 @@ func (c *StorageClient) RuntimeEvents(ctx context.Context, p apiTypes.GetRuntime
 	return &es, nil
 }
 
+// Fetch account balances from the node and pass them back via the channel. Note that if the input context
+// times out or we encounter an error, we explicitly close the channel and return. This enables the calling
+// function to create a context with a timeout and then listen to the channel with the expectation that the
+// channel will return values and/or close approximately within the timeout.
+func (c *StorageClient) fetchAccountBalanceFromNode(ctx context.Context, ch chan *RuntimeSdkBalance, runtime common.Runtime, address staking.Address) {
+	var latestIndexedHeight uint64
+	// Query latest block.
+	err := c.db.QueryRow(
```

Comment:
does our library allow concurrent queries from one shared db object? might want to query this outside the goroutine if we're not sure

Reply:
think so; `http.Server` is already serving queries concurrently using the same db object, and the analyzers share a db object as well.